sktime/forecasting/trend/_polynomial_trend_forecaster.py

```diff
@@ -18,28 +18,33 @@
 class PolynomialTrendForecaster(BaseForecaster):
     r"""Forecast time series data with a polynomial trend.
 
-    Uses a `sklearn` regressor specified by the `regressor` parameter
+    Uses an ``sklearn`` regressor specified by the ``regressor`` parameter
     to perform regression on time series values against their corresponding indices,
     after extraction of polynomial features.
-    Same as `TrendForecaster` where `regressor` is pipelined with transformation step
-    `PolynomialFeatures(degree, with_intercept)` applied to time index, at the start.
+    Same as ``TrendForecaster`` where ``regressor`` is pipelined with transformation
+    step ``PolynomialFeatures(degree, with_intercept)`` applied to time index,
+    at the start.
 
-    In `fit`, for input time series :math:`(v_i, p(t_i)), i = 1, \dots, T`,
+    In ``fit``, for input time series :math:`(v_i, p(t_i)), i = 1, \dots, T`,
     where :math:`v_i` are values, :math:`t_i` are time stamps,
     and :math:`p` is the polynomial feature transform with degree `degree`,
     and with/without intercept depending on `with_intercept`,
-    fits an `sklearn` model :math:`v_i = f(p(t_i)) + \epsilon_i`, where `f` is
-    the model fitted when `regressor.fit` is passed `X` = vector of :math:`p(t_i)`,
-    and `y` = vector of :math:`v_i`.
+    fits an `sklearn` model :math:`v_i = f(p(t_i)) + \epsilon_i`, where :math:`f` is
+    the model fitted when ``regressor.fit`` is passed ``X`` = vector of :math:`p(t_i)`,
+    and ``y`` = vector of :math:`v_i`.
 
-    In `predict`, for a new time point :math:`t_*`, predicts :math:`f(p(t_*))`,
-    where :math:`f` is the function as fitted above in `fit`,
+    In ``predict``, for a new time point :math:`t_*`, predicts :math:`f(p(t_*))`,
+    where :math:`f` is the function as fitted above in ``fit``,
     and :math:`p` is the same polynomial feature transform as above.
 
-    Default for `regressor` is linear regression = `sklearn` `LinearRegression` default.
+    Default for ``regressor`` is linear regression = ``sklearn`` ``LinearRegression``,
+    with default parameters. Default for `degree` is 1.
+
+    is polynomial regression, obtained as
+    ``sklearn`` ``LinearRegression`` default.
```

Comment:
The line split here looks a bit odd, does it not fit?

Reply:
Are these two lines (L43-L44) required? It seems to be covered in L40-L41 already.

![image](https://github.com/sktime/sktime/assets/39331844/ce64b8a3-d59c-46bd-ac17-57ef4d88e1b0)

Reply:
> The line split here looks a bit odd, does it not fit?

Otherwise it goes over the margin - it does not appear as a line split in the rendered docs though.

Reply:
> Are these two lines (L43-L44) required?

Looks like an accident, fixing.